### ISSUE TITLE
Microsoft Outlook date picker is not accessible with NVDA

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#7217]
### Steps to reproduce:
With NVDA running:
1. Start Microsoft Outlook 2016
2. Create an appointment (Home -> New Items -> Appointment)
3. In the Appointment dialog, tab to the date picker button and press space bar to activate it.
4. Press tab or the arrows 1 or more times to move focus around the available controls in the date picker.
5. Notice that NVDA announces nothing at all.

### Expected results
NVDA should announce each control as it gets focus.

### Technical info
These controls are implemented as one MSAA table cell object, in a window completely disassociated with the date picker button and or the dialog. GainFocus events are fired, but always using exactly the same event parameters. Thus:
1. the Outlook appModule will have to allow all gainFocus events through for that particular window class using requestEvents, and the datePciker cell object will have to no longer ignore duplicate IAccessible events.
